app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PiecesController < ApplicationController
+	def show
+    @piece = Piece.find(params[:id])
+    @pieces = @piece.game.pieces # display all the pieces on the board
+  end
+
+  private
+
+  def piece_params
+    params.require(:piece).permit(:pos_x, :pos_y)
```

Comment:
You'll probably want to pass in :end_pos_x and :end_pos_y just to avoid confusion with the existing start position...